R/periscope2.R

```diff
@@ -72,3 +72,10 @@ NULL
         }
     }
 }
+
+
+.onLoad <- function(libname, pkgname) {
+    if ("package:periscope" %in% search()) {
```

Comment:
Wrap in a try/catch and put out a console warning if it cannot be unloaded

Reply:
done